test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,24 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        Object nullObj = null;
```

Comment:
This line isn't necessary. You can just consider using the value `null` directly in the methods below.